src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -46,13 +47,6 @@ interface FeedbackQuestionsResponse {
   questions: FeedbackQuestion[];
 }
 
-interface FeedbackQuestionRecipients {
-  recipients: {
```

Comment:
Can you check if there are any place reference `{
    name: string,
    identifier: string,
  }` and change it to `FeedbackQuestionRecipient` accordingly?